mlflow/server/js/src/components/CompareRunScatter.css

```diff
@@ -1,5 +1,5 @@
 .scatter-tooltip {
-  width: 300px;
+  width: 340px;
```

Comment:
Increased the width of the tooltip to support showing run names of the form "Run <uuid>"